app/Views/errors/html/error_exception.php

```diff
@@ -19,7 +19,7 @@
 	<!-- Header -->
 	<div class="header">
 		<div class="container">
-			<h1><?= esc($title), esc($exception->getCode() ? ' #' . $exception->getCode() : '') ?></h1>
+			<h1><?= esc($title) ?></h1>
```

Comment:
Why do you remove the exception code?

Reply:
This was made to make the output similar to its CLI equivalent.

Reply:
Should have been a separate PR, I guess?